src/components/SweetsList/CandyItemStyles.ts

```diff
@@ -4,13 +4,29 @@ import { Container, Item } from "../../common/styles/componentsStyles";
 
 const StyledCandyItem = styled(Item)`
   gap: 1rem;
+  //FIXME:
+  /* z-index: auto; */
 
   & .item__details {
     display: flex;
     flex-direction: column;
     gap: 0.5rem;
     text-align: left;
     width: 65%;
+
+    @media (max-width: 710px) {
```

Comment:
it's better when all the media queries are consistent and coming from your theme